src/main/java/blackjack/domain/Card.java

```diff
@@ -0,0 +1,27 @@
+package blackjack.domain;
+
+public class Card {
+    private final CardNumber cardNumber;
+    private final CardSymbol cardSymbol;
+
+    public Card(CardNumber cardNumber, CardSymbol cardSymbol) {
+        this.cardNumber = cardNumber;
+        this.cardSymbol = cardSymbol;
+    }
+
+    public boolean isACE() {
+        return cardNumber.number().equals(CardNumber.ACE.number());
+    }
+
+    public int getScore() {
+        return CardNumber.scoreByNumber(cardNumber.number());
```

Comment:
```suggestion
        return cardNumber.score();
```
score() 메서드를 이용하면 되지 않을까요?
그리고 전체적으로 getXXX 라는 네이밍을 많이 사용하셨던데 통일하면 좋을 것 같습니다.